tests/end2end/features/history.feature

```diff
@@ -77,7 +77,7 @@ Feature: Page history
     Scenario: Listing history
         When I open data/numbers/3.txt
         And I open data/numbers/4.txt
-        And I open qute://history/data
+        And I open qute://history
         Then the page should contain the plaintext "3.txt"
         Then the page should contain the plaintext "4.txt"
```

Comment:
Probably webkit preventing JS from accessing qute:history/data, but allowing it to be accessed directly.